### PR TITLE
default these env vars for specs

### PR DESCRIPTION
**Pull request by curtis-rainforestqa**
so we don't have to explicitly define them to
run the suite.

### CODE REVIEW

File: lib/simple_sqs.rb [hunk 1]
```diff
@@ -7,6 +7,7 @@
 require "sentry-raven"
 require "librato-rails"
 require "aws-sdk"
+require 'multi_json'
```
Comment (epaulet) [inline lib/simple_sqs.rb:10]: Is this related?

Comment (curtis-rainforestqa) [reply lib/simple_sqs.rb:10]: yes, intentional add. the specs will fail without it.

**Review by epaulet — APPROVED** [review]
